powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/CallbackUrlEntity.java

```diff
@@ -139,6 +139,18 @@ public class CallbackUrlEntity implements Serializable {
     @Column(name = "enabled", nullable = false)
     private boolean enabled = true;
 
+    /**
+     * Timestamp of the creation.
+     */
+    @Column(name = "timestamp_created", nullable = false, columnDefinition = "TIMESTAMP DEFAULT CURRENT_TIMESTAMP")
+    private LocalDateTime timestampCreated = LocalDateTime.now();
+
+    /**
+     * Timestamp of the last update via Callback Management API.
```

Comment:
While this is true right now, later on, it could be updated from different places so the JavaDoc is probably too strict.
```suggestion
     * Timestamp of the last update.
```